### ISSUE TITLE
how did you do model selection/validation?

### ISSUE THREAD

**Issue by rguo12** [deepmind/conformal_training#3]
Hi David,

Reading the paper, I find the Table A, there is no validation set, only training, calibration and test. I wonder how did you do model selection in this case? I also skimmed over the code, seems you merged validation and test and then randomly split them and use the new validation as the calibration. Not sure if there is a validation set.

Best,

RC

**Comment by davidstutz** [comment]
Good question. We actually merged validation and calibration set. Meaning we selected hyper-parameters on the calibration set. Strictly speaking, you would ideally have a separate validation and calibration set (+ training and test sets). We found it acceptable to merge both for simplicity.

**Comment by rguo12** [comment]
Thanks David, by 'acceptable', do you mean using the same set for validation and calibration does not impact the coverage rate / prediction set size?

**Comment by davidstutz** [comment]
Yes, we saw insignificant impact in terms of coverage rate or inefficiency.
